src/main/java/com/bookstore/bookstoreapi/data/entity/Customer.java

```diff
@@ -42,6 +42,11 @@ public class Customer {
   @Column(name = "Address", nullable = false)
   private String address;
 
+  @EqualsAndHashCode.Exclude
+  @OneToMany(mappedBy = "customer", cascade = CascadeType.ALL)
+  @Builder.Default
+  private Set<Comment> comments = new HashSet<>();
```

Comment:
> Why is this a set?

would it not be the same reason ratings is a set? since the user can have multiple comments like they have multiple ratings? im just going off what works here

Reply:
I would say that multiple ratings are not possible, but multiple comments are, but idk if that makes sense.